packages/manager/modules/vps/src/dashboard/tile/configuration/upgrade/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,11 @@
+{
+  "vps_dashboard_tile_configuration_upgrade_memory_action_title": "Améliorer la mémoire de votre VPS de 2 à 4 Go",
+  "vps_dashboard_tile_configuration_upgrade_memory_action_details": "Vous êtes sur le point d‘ajouter {{ memoryQuantity }} de mémoire à votre VPS.",
+  "vps_dashboard_tile_configuration_upgrade_storage_action_title": "Améliorer le stockage de votre VPS de 40 à 80 Go",
```

Comment:
Have we got any ways to get those values from API ? Because I'm wondering if the values can change like go from 80 Go to 100 Go for instance